### PR TITLE
[REF] web_tour: decrease default checkDelay

### PR DESCRIPTION
**Pull request by pipu-odoo**
By default, the checkdelay is 500ms when it is not stipulated in a tour. For performance reasons, we reduce this delay to 400ms.

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by BastienFafchamps** [comment]
@robodoo r+

**Comment by BastienFafchamps** [comment]
@robodoo retry

**Comment by BastienFafchamps** [comment]
@robodoo r+

**Comment by BastienFafchamps** [comment]
@robodoo r+

**Comment by BastienFafchamps** [comment]
@robodoo r+

File: addons/l10n_ec_website_sale/static/tests/tours/website_sale_checkout_address.js [hunk 1]
```diff
@@ -46,19 +46,17 @@ registry.category("web_tour.tours").add("tour_new_billing_ec", {
...
-            trigger: '.all_billing a[href^="/shop/address?address_type=billing"]:contains("Add address")',
+            trigger: `.all_billing a[href^="/shop/address?address_type=billing"]:contains(Add address)`,
             run: "click",
         },
         ...tourUtils.fillAdressForm(),
         {
-            content: "Save address",
-            trigger: "button#save_address",
-            run: "click",
+            trigger: `[name="address_card"] address:contains(1 rue de la paix)`,
```
Comment (pipu-odoo) [inline addons/l10n_ec_website_sale/static/tests/tours/website_sale_checkout_address.js:59]: This has changed from your latest review

**Comment by BastienFafchamps** [comment]
@robodoo r+

**Comment by pipu-odoo** [comment]
@robodoo r- ... still error in l10n tour :'(

**Comment by pipu-odoo** [comment]
@robodoo r-
... still error in l10n tour :'(

**Review by aab-odoo — COMMENTED** [review]
robodoo r+

**Pull request closed without merging**